Assets/Scripts/WorldEngine/Terrain/TerrainCell.cs

```diff
@@ -287,21 +288,6 @@ public void SetAlteration(TerrainCellAlteration alteration)
 
         BaseTemperatureOffset = alteration.BaseTemperatureOffset;
         BaseRainfallOffset = alteration.BaseRainfallOffset;
-        
-        OriginalAltitude = alteration.OriginalAltitude;
-        Altitude = alteration.Altitude;
-
-#if DEBUG
-        if (!alteration.Temperature.IsInsideRange(World.MinPossibleTemperatureWithOffset - 0.5f, World.MaxPossibleTemperatureWithOffset + 0.5f))
-        {
-            Debug.Log("Invalid alteration.Temperature: " + alteration.Temperature);
-        }
-#endif
-
-        Temperature = alteration.Temperature;
-        OriginalTemperature = alteration.OriginalTemperature;
-        Rainfall = alteration.Rainfall;
-        WaterAccumulation = alteration.WaterAccumulation;
```

Comment:
Why were these variables removed? Not used?

Reply:
They were used before. Originally, when a cell was edited in the editor, the changes would be saved to file in a "TileCellAlteration" object. But I found out that these changes where not being calculated correctly when using brushes and it was better to regenerate them globally once the brush became inactive, or after a file load. The only real values that need to be stored in file if changed are the ones prefixed with "Base..." The others can be derived from these.